kerl

```diff
@@ -782,9 +818,27 @@ do_active()
 
 download()
 {
+    mkdir -p "$KERL_DOWNLOAD_DIR"
+    if [ "$KERL_BUILD_BACKEND" == "git" ]; then
+        FILENAME="OTP-$1"
+        github_download "$FILENAME.tar.gz"
+    else
+        FILENAME="otp_src_$1"
+        tarball_download "$FILENAME.tar.gz"
+    fi
+}
+
+github_download()
+{
     if [ ! -f "$KERL_DOWNLOAD_DIR/$1" ]; then
         echo "Downloading $1 to $KERL_DOWNLOAD_DIR"
-        mkdir -p "$KERL_DOWNLOAD_DIR"
+        curl -L "$OTP_GITHUB_URL/archive/$1" > "$KERL_DOWNLOAD_DIR/$1"
```

Comment:
Thinking about making this otp_src_$1 too just so its standardized across both download backends